java-agent/src/nativeMain/kotlin/com/epam/drill/agent/Agent.kt

```diff
@@ -57,6 +57,7 @@ object Agent {
         println(LOGO)
         try {
             defaultNativeLoggingConfiguration()
+            drillInstallationDir = getDrillInstallationDir()
```

Comment:
declare either `var` or `val` identifier (its never reassigned so `val` I guess)

Reply:
Moved to agent lib, so now it's possible to use just val w/o atomics.